packages/react/src/drawer-trigger/DrawerTrigger.tsx

```diff
@@ -0,0 +1,18 @@
+import * as RadixDialog from "@radix-ui/react-dialog";
```

Comment:
Vai, should we keep it as RadixDrawer like other Drawer components?

Reply:
No, why did we use RadixDrawer in other places? The package and component is dialog and should be called that to prevent confusion.

Reply:
Then, we need to change where we declared this as `RadixDrawer` vai. Previously maintained the alias, as per our component naming.

Reply:
addressed